### PR TITLE
Feature 1894/combine work preference and computer accessories

### PR DESCRIPTION
**Pull request by BenVuong**

### CODE REVIEW

File: onboard/src/pages/EquipmentPage.jsx [hunk 1]
```diff
@@ -42,6 +47,70 @@ const EquipmentPage = () => {
       direction="row"
     >
       <Grid item xs={8}>
+      <h2 style={{ marginTop: "1rem" }}>
+          Please select where you would perfer to work:
```
Comment (MisheelSodgerel) [inline onboard/src/pages/EquipmentPage.jsx:51]: *prefer

**Review by MisheelSodgerel — COMMENTED** [review]
small typo

File: onboard/src/pages/WebPortal.jsx [hunk 2]
```diff
@@ -29,38 +29,30 @@ const menuList = [
...
-    completed: false,
-    child: <WorkingLocationPage />,
-    visited: false,
-  },
-  {
-    index: 3,
-    name: "Computer and Accessories",
-    title: "Computer and Accessories",
+    name: "Computer & Work Preferences",
+    title: "Computer & Work Preference",
```
Comment (danie1r) [inline onboard/src/pages/WebPortal.jsx:33]: The page name should be "Work Environment"

File: onboard/src/pages/WebPortal.jsx [hunk 3]
```diff
@@ -29,38 +29,30 @@ const menuList = [
...
     completed: false,
     child: <EquipmentPage />,
```
Comment (danie1r) [inline onboard/src/pages/WebPortal.jsx:35]: Maybe consider changing the component name as well to prevent confusion in the future

**Review by 1mbalducci — APPROVED** [review]

**Review by S78901 — APPROVED** [review]

**Review by MatthewBackes — APPROVED** [review]

**Pull request merged**